webpack/assets/javascripts/react_app/components/MemoryAllocationInput/MemoryAllocationInput.js

```diff
@@ -105,14 +88,17 @@ const MemoryAllocationInput = ({
     <Grid>
       <Row>
         <Col>
-          <NumericInput
+          <ForemanNumericInput
             value={value}
+            rawValue={value * MEGABYTES}
+            name={name}
+            id={id}
             format={format}
             parser={parser}
             onChange={handleChange}
             label={label}
-            name={name}
             disabled={disabled}
+            minValue={minValue}
           />
```

Comment:
Couldn't we just add the hidden input with raw value here instead of having completely new component for it?
We would just not pass the `name` prop down to the NumericInput and thus the underlying component.

Reply:
@ezr-ondrej  So just to see that I understood what you are saying, you are suggesting to put here the hidden value and then there will be no need for the ForemanNumericInput and I could use the NumericInput directly?

Reply:
yup exactly! :)
Unless I'm missing another reason we are introducing the `ForemanNumericInput`?

Reply:
I think this is the only reason and I agree with you, I will change this.